unimport/constants.py

```diff
@@ -6,9 +6,8 @@
 VERSION = "0.3.0"
 
 PY38_PLUS = sys.version_info >= (3, 8)
-PY37_PLUS = sys.version_info >= (3, 7)
 
-SUBSCRIPT_TYPE_VARIABLE = [
+SUBSCRIPT_TYPE_VARIABLE = {
```

Comment:
Hi, thank you, Here we can make it more efficient using frozenset.

```suggestion
SUBSCRIPT_TYPE_VARIABLE = frozenset({
```

Reply:
Yeah, you're right!

Reply:
I've changed it to `frozenset`, but to be honest, there's no difference between the normal `set` efficiency and the `frozenset`!

Reply:
My pleasure... you're right, changed!